scheduler/manifests/install/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+namespace: keptn-lifecycle-controller-system
```

Comment:
the indentation is not preserved with the deleted file
https://github.com/keptn-sandbox/lifecycle-controller/pull/84/files#diff-8ee8698388b5698f0c2ffc57107eaff9e22e64983ec721511effd427de3ff4ff